src/common.rs

```diff
@@ -5,6 +5,10 @@
 
 #![allow(missing_docs)]
 
+mod common_sealed {
```

Comment:
I'm not sure that `common_sealed` is a great name for this - I think I'd prefer something like:
```rust
mod sealed {
    /// Seals [super::FloatFunc]
    pub trait FloatFuncsSealed {}
}
```

I don't think the Rust project documents sealed traits itself, so I think we should also probably have a link which explains it.
I don't know what link that would be, however

Reply:
fixed in [cb3c29d](https://github.com/linebender/kurbo/pull/332/commits/cb3c29d41a0163f3118954e4c0ef44a0b53023f2)